model/telemetry/metrics.yaml

```diff
@@ -0,0 +1,98 @@
+groups:
+  - id: metric.telemetry.sdk.span.ended_count
+    type: metric
+    metric_name: telemetry.sdk.span.ended_count
+    stability: experimental
+    brief: "The number of spans which have been ended"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.span.sampled_count
+    type: metric
+    metric_name: telemetry.sdk.span.sampled_count
+    stability: experimental
+    brief: "The number of spans which have been ended AND are sampled"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.span.processor.queue_size
+    type: metric
+    metric_name: telemetry.sdk.span.processor.queue_size
+    stability: experimental
+    brief: "The number of spans in the queue of a given instance of an SDK span processor"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: updowncounter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.span.processor.queue_capacity
+    type: metric
+    metric_name: telemetry.sdk.span.processor.queue_capacity
+    stability: experimental
+    brief: "The maximum number of spans the queue of a given instance of an SDK span processor can hold"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: gauge
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.span.processor.spans_submitted
```

Comment:
This is going to be identical of the sampled spans for all processors today.  We should probably make one of the two metrics optional and opt-in.

Reply:
I support removing `spans_submitted`.  @jmacd It could differ if we introduced a mechanism to have chained/nested processors.  But I don't think it is necessary today. I think you mean spans_sampled, rather than spans_ended?  Seems like spans_ended includes non-sampled spans as well.

Reply:
spans_submitted is removed in the latest version.